tableauserverclient/server/request_factory.py

```diff
@@ -129,6 +129,54 @@ def create_req(self, project_item):
         return ET.tostring(xml_request)
 
 
+class ScheduleRequest(object):
+    def create_req(self, schedule_item):
+        xml_request = ET.Element('tsRequest')
+        schedule_element = ET.SubElement(xml_request, 'schedule')
+        schedule_element.attrib['name'] = schedule_item.name
+        schedule_element.attrib['priority'] = str(schedule_item.priority)
+        schedule_element.attrib['type'] = schedule_item.schedule_type
+        schedule_element.attrib['executionOrder'] = schedule_item.execution_order
+        interval_item = schedule_item.interval_item
+        schedule_element.attrib['frequency'] = interval_item.frequency
+        frequency_element = ET.SubElement(schedule_element, 'frequencyDetails')
+        frequency_element.attrib['start'] = str(interval_item.start_time)
+        if interval_item.end_time:
+            frequency_element.attrib['end'] = str(interval_item.end_time)
+        if interval_item.interval:
+            intervals_element = ET.SubElement(frequency_element, 'intervals')
+            for interval in interval_item.interval:
+                (expression, value) = interval
+                single_interval_element = ET.SubElement(intervals_element, 'interval')
+                single_interval_element.attrib[expression] = value
+        return ET.tostring(xml_request)
+
+    def update_req(self, schedule_item):
+        xml_request = ET.Element('tsRequest')
+        schedule_element = ET.SubElement(xml_request, 'schedule')
+        if schedule_item.name:
```

Comment:
This could also be turned in a loop across all the different attributes.  since it's pretty prescriptive, the only tricky-ish part is the conversion of things like executionOrder to execution_order, but I think that's well understood.